test/e2e/v1beta1/scripts/gh-actions/setup-katib.sh

```diff
@@ -65,10 +65,24 @@ echo "Deploying Katib"
 cd ../../../../../ && WITH_DATABASE_TYPE=$WITH_DATABASE_TYPE make deploy && cd -
 
 # Wait until all Katib pods is running.
+MAX_RETRY_COUNT=10
+until [ "${MAX_RETRY_COUNT}" = "0" ] || [ "$(kubectl get pods -n kubeflow -l "katib.kubeflow.org/component in ($WITH_DATABASE_TYPE,controller,ui,db-manager)" -oname | wc -l)" = "4" ]; do
+  echo "Waiting for katib components to be created"
+  sleep 2
+  MAX_RETRY_COUNT=$((MAX_RETRY_COUNT - 1))
+done
```

Comment:
To avoid the following error:

```shell
error: no matching resources found
No resources found in kubeflow namespace.
No resources found in kubeflow namespace.
Error: Process completed with exit code 1.
```

https://github.com/kubeflow/katib/actions/runs/5842762639/job/15852810097?pr=2201#step:4:5303

Reply:
This will be fixed by https://github.com/kubeflow/katib/pull/2203.